src/test/java/net/snowflake/ingest/streaming/internal/ParquetValueParserTest.java

```diff
@@ -0,0 +1,136 @@
+package net.snowflake.ingest.streaming.internal;
+
+import java.math.BigInteger;
+import org.apache.parquet.schema.PrimitiveType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ParquetValueParserTest {
+
+  @Test
+  public void parseValueINT32() {
+    // FIXED, SB1
+    ColumnMetadata testCol =
+        ColumnMetadataBuilder.newBuilder()
+            .physicalType("SB1")
+            .logicalType("FIXED")
+            .nullable(true)
+            .build();
+
+    RowBufferStats rowBufferStats = new RowBufferStats();
+    ParquetValueParser.ParquetBufferValue pv =
+        ParquetValueParser.parseColumnValueToParquet(
+            123, testCol, PrimitiveType.PrimitiveTypeName.INT32, rowBufferStats);
+
+    Assert.assertEquals(Integer.class, pv.getValue().getClass());
+    Assert.assertEquals(4.0, pv.getSize(), 0);
+    Assert.assertEquals(BigInteger.valueOf(123), rowBufferStats.getCurrentMinIntValue());
+    Assert.assertEquals(BigInteger.valueOf(123), rowBufferStats.getCurrentMaxIntValue());
+  }
+
+  @Test
+  public void parseValueINT64() {
+    ColumnMetadata testCol =
+        ColumnMetadataBuilder.newBuilder()
+            .physicalType("SB8")
+            .logicalType("FIXED")
```

Comment:
Many parquet types are used to pack different SF logical types,
e.g. for INT64: FIXED, TIME, TIMESTAMP_*
BINARY is also used for OBJECT and VARIANT/ARRAY
some of those logical types are very different and each deserves its own test

Reply:
@sfc-gh-azagrebin I added other tests. Please take a look and tell me if I missed any important cases.